### PR TITLE
Multi-language support with localization

### PR DESCRIPTION
**Pull request by byrro**
Noticed [PR#15](https://github.com/Ashark/davinci-resolve-checker/pull/15) and agree it'd be a maintenance nightmare

This PR is a suggested implementation for localization. Started with samples for zh_CN and pt_BR.

I localized a few strings in the main script as a sample. If you think this implementation is fine, I'll localize the rest of the script for zh_CN and pt_BR. Others could later contribute additional languages.

### CODE REVIEW

**Comment by Ashark** [comment]
Yeah! Thank you very much. I wanted such solution when you wrap strings to function from the main script.
Please, proceed with the rest.
Can you also fix your language files to include [end new line](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)?

**Comment by Ashark** [comment]
Thank you very much!
One suggestion. Is it also possible to translate the chassis type value? I.e. that "laptop" or "desktop" strings.

**Comment by Ashark** [comment]
Thanks. There is a minor problem with spaces. It currently loks like this for me:
```
Presented GPUs:
        CometLake-S GT2 [UHD Graphics 630] (kernel driver in use:i915)
        Ellesmere [Radeon RX 470/480/570/570X/580/580X/590] (kernel driver in use:amdgpu)
        GK208 [GeForce GT 630 Rev. 2] (kernel driver in use:vfio-pci)
```
Could you please preserve those spaces before "use:"?

File: davinci-resolve-checker.py [hunk 1]
```diff
@@ -35,22 +51,22 @@
...
 # lspci -d ::0380 - amd secondary gpu on an i+a laptop
 
-print("Presented GPUs:")
-print ("\t" + "\n\t".join([ x.device.name + " (kernel driver in use: " + x.driver + ")" for x in lspci_devices if x.cls.id in (0x0300, 0x0301, 0x0302, 0x0380) ]))
+print(local_str["presented gpus"])
+print ("\t" + "\n\t".join([ x.device.name + " (" + local_str["kernel driver"] + x.driver + ")" for x in lspci_devices if x.cls.id in (0x0300, 0x0301, 0x0302, 0x0380) ]))
 
 GL_VENDOR = subprocess.check_output('glxinfo | grep "OpenGL vendor string" | cut -f2 -d":"', shell=True, text=True).strip()
-print("OpenGL vendor string: " + GL_VENDOR)
+print(local_str["opengl vendor"], GL_VENDOR)
```
[the remark this answers is not in the document]
Comment (byrro) [reply davinci-resolve-checker.py:58]: In this case, the `print` statement will already add space between the arguments

File: davinci-resolve-checker.py [hunk 2]
```diff
@@ -35,22 +51,22 @@
 # lspci -d ::0302 - nvidia 3d controller on an optimus laptop
 # lspci -d ::0380 - amd secondary gpu on an i+a laptop
 
-print("Presented GPUs:")
-print ("\t" + "\n\t".join([ x.device.name + " (kernel driver in use: " + x.driver + ")" for x in lspci_devices if x.cls.id in (0x0300, 0x0301, 0x0302, 0x0380) ]))
+print(local_str["presented gpus"])
+print ("\t" + "\n\t".join([ x.device.name + " (" + local_str["kernel driver"] + x.driver + ")" for x in lspci_devices if x.cls.id in (0x0300, 0x0301, 0x0302, 0x0380) ]))
```
[the remark this answers is not in the document]
Comment (byrro) [reply davinci-resolve-checker.py]: Sure, need an explicit space here indeed, just added it

**Pull request merged**